docs/instrumentation-list.yaml

```diff
@@ -0,0 +1,1603 @@
+      JAVAAGENT: []
+  - name: opentelemetry-extension-annotations-1.0
+    srcPath: instrumentation/opentelemetry-extension-annotations-1.0
+    target_versions:
+      JAVAAGENT:
+      - io.opentelemetry:opentelemetry-extension-annotations:[0.16.0,)
+  - name: opentelemetry-instrumentation-annotations-1.16
+    srcPath: instrumentation/opentelemetry-instrumentation-annotations-1.16
+    target_versions:
+      JAVAAGENT:
+      - io.opentelemetry:opentelemetry-instrumentation-annotations:(,)
+  - name: opentelemetry-instrumentation-api
+    srcPath: instrumentation/opentelemetry-instrumentation-api
+    target_versions:
+      JAVAAGENT:
+      - io.opentelemetry.instrumentation:opentelemetry-instrumentation-api:[1.14.0-alpha,)
+  - name: opentelemetry-api-1.37
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.37
+    target_versions:
+      JAVAAGENT: []
+  - name: opentelemetry-api-1.38
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.38
+    target_versions:
+      JAVAAGENT: []
+  - name: opentelemetry-api-1.31
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.31
+    target_versions:
+      JAVAAGENT: []
+  - name: opentelemetry-api-1.32
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.32
+    target_versions:
+      JAVAAGENT: []
+  - name: opentelemetry-api-1.42
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.42
+    target_versions:
+      JAVAAGENT: []
+  - name: opentelemetry-api-1.40
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.40
+    target_versions:
+      JAVAAGENT: []
+  - name: opentelemetry-api-1.47
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.47
+    target_versions:
+      JAVAAGENT: []
+  - name: opentelemetry
+    srcPath: instrumentation/wicket-8.0/common-testing/src/main/java/io/opentelemetry
+    target_versions: {}
+  - name: opentelemetry-extension-kotlin-1.0
+    srcPath: instrumentation/opentelemetry-extension-kotlin-1.0
+    target_versions:
+      JAVAAGENT:
+      - io.opentelemetry:opentelemetry-extension-kotlin:[0.17.0,)
+  - name: opentelemetry-api-1.4
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.4
+    target_versions:
+      JAVAAGENT: []
+  - name: opentelemetry-api-1.0
+    srcPath: instrumentation/opentelemetry-api/opentelemetry-api-1.0
+    target_versions:
+      JAVAAGENT:
+      - io.opentelemetry:opentelemetry-api:[0.17.0,)
+oracle:
+  instrumentations:
+  - name: oracle-ucp-11.2
+    srcPath: instrumentation/oracle-ucp-11.2
+    target_versions:
+      JAVAAGENT:
+      - com.oracle.database.jdbc:ucp:[,)
+      LIBRARY:
+      - com.oracle.database.jdbc:ucp:11.2.0.4
+      - com.oracle.database.jdbc:ojdbc8:12.2.0.1
+oshi:
+  instrumentations:
+  - name: oshi
+    srcPath: instrumentation/oshi
+    target_versions:
+      JAVAAGENT:
+      - com.github.oshi:oshi-core:[5.3.1,)
+      LIBRARY:
+      - com.github.oshi:oshi-core:$oshiVersion
```

Comment:
I'd be in favor of just bumping oshi min version to 5.5.0 and getting rid of the conditional for arm mac, it's still reasonably old version and I don't think it's a very popular instrumentation